packages/clerk-js/src/ui/components/OrganizationProfile/OrganizationProfileRoutes.tsx

```diff
@@ -12,65 +14,87 @@ import { VerifiedDomainPage } from './VerifiedDomainPage';
 import { VerifyDomainPage } from './VerifyDomainPage';
 
 export const OrganizationProfileRoutes = (props: PropsOfComponent<typeof ProfileCardContent>) => {
+  const { pages } = useOrganizationProfileContext();
+  const isMembersPageRoot = pages.routes[0].id === 'members';
```

Comment:
Any chance this array is empty?

Reply:
No chance; it always appends Clerk default routes.